scripts/scr_weapon/scr_weapon.gml

```diff
@@ -184,7 +184,7 @@ global.weapons = {
 		"range": 1,
 		"spli": 2,
 		"arp": 0,
-		"tags": ["sword"],
+		"tags": ["blade"],
```

Comment:
I don't remember if it's the case for other tags, like axe or knife, but iirc sword tag was checked for duelist trait damage bonus.

Reply:
I'll adjust it thank you for the insight